docs/components/camera/image-file.md

```diff
@@ -66,8 +67,9 @@ The following attributes are available for `image_file` cameras:
 | `intrinsic_parameters` | object | Optional | The intrinsic parameters of the camera used to do 2D <-> 3D projections: <ul> <li> <code>width_px</code>: The expected width of the aligned image in pixels. </li> <li> <code>height_px</code>: The expected height of the aligned image in pixels. </li> <li> <code>fx</code>: The image center x point. </li> <li> <code>fy</code>: The image center y point. </li> <li> <code>ppx</code>: The image focal x. </li> <li> <code>ppy</code>: The image focal y. </li> </ul> |
 | `distortion_parameters` | object | Optional | Modified Brown-Conrady parameters used to correct for distortions caused by the shape of the camera lens: <ul> <li> <code>rk1</code>: The radial distortion x. </li> <li> <code>rk2</code>: The radial distortion y. </li> <li> <code>rk3</code>: The radial distortion z. </li> <li> <code>tp1</code>: The tangential distortion x. </li> <li> <code>tp2</code>: The tangential distortion y. </li> </ul> |
 | `debug` | boolean | Optional | Enables the debug outputs from the camera if `true`. <br> Default: `false` |
-| `color_image_file_path` | string | Optional | The file path to the color image. |
-| `depth_image_file_path` | string | Optional | The file path to the depth image. |
+| `color_image_file_path` | string | Optional | The file path to the color image on your local system. |
+| `depth_image_file_path` | string | Optional | The file path to the depth image on your local system. |
+| `pointcloud_file_path` | string | Optional | The file path to the point cloud file on your local system. You must specify `depth_image_file_path` to use `pointcloud_file_path`. |
```

Comment:
It's actually just necessary to have one of the three `file_path` set. You can also just have `pointcloud_file_path` without the other two files paths set.

Reply:
Ok, cool! In that case I am also going to remove this usage note at the top of the page: 

```
If you provide a depth image, as well as the intrinsic parameters of the camera that took that depth image, then the camera will also provide a point cloud.
```
Assuming this is now inaccurate and this attribute simplifies usage?